src/repositorios/colllection.test.js

```diff
@@ -0,0 +1,162 @@
+"use strict";
+
+jest.useFakeTimers();
+
+let CollectionDOA = require("./collection");
+
+
+describe(
+    "Test CollectionModel",
+    () => {
+
+        /*
+            Test For Create Collection 
+        */
+
+        test(
+            "create_collection",
+            () => {
+                return CollectionDOA.create_collection({
+                    name: "Jest",
+                    slang: "jest",
+                    public: true,
+                    description: "test with jest",
+                    owner: 1
+                })
+                .then(
+                    data => {
+                        expecte(data)
```

Comment:
This will fail because Sequelize is configured to return a query object instead of JSON. You need to use `data.dataValues`.